llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -1378,6 +1378,20 @@ void State::addInfoForInductions(BasicBlock &BB) {
 void State::addInfoFor(BasicBlock &BB) {
   addInfoForInductions(BB);
 
+  auto addConditionFact = [&](DomTreeNode *DTN, CmpInst::Predicate Pred,
```

Comment:
nit: variable names usually start with capital letter
```suggestion
  auto AddConditionFact = [&](DomTreeNode *DTN, CmpInst::Predicate Pred,
```

Reply:
Oh, good point. In my mind I was thinking of this as a function.